<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="controlFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statsButton">
         <property name="text">
          <string>Show Stats</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algorithmComboBox">
         <item>
          <property name="text">
           <string>Bubble Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Selection Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Insertion Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quick Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Merge Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shell Sort</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear Search</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Binary Search</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="findEdit">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sizeComboBox">
         <item>
          <property name="text">
           <string>10 Elements</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100 Elements</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000 Elements</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="statsLabel">
         <property name="text">
          <string>Swaps: 0 Comparisons: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="speedLabel">
         <property name="text">
          <string>delay ms: 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="speedSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
